osu.Framework/Bindables/Bindable.cs

```diff
@@ -295,7 +296,7 @@ public virtual void Parse([CanBeNull] object input)
                     if (underlyingType.IsEnum)
                         Value = (T)Enum.Parse(underlyingType, input.ToString().AsNonNull());
                     else
-                        Value = (T)Convert.ChangeType(input, underlyingType, CultureInfo.InvariantCulture);
+                        Value = (T)Convert.ChangeType(input, underlyingType, cultureInfo ?? CultureInfo.InvariantCulture);
```

Comment:
```suggestion
                        Value = (T)Convert.ChangeType(input, underlyingType, cultureInfo ?? CultureInfo.CurrentCulture);
```

Using the invariant culture breaks my expectation that `ToString()` and `Parse()` are opposites to each other. 
The following test illustrates that:

```cs
[TestCase(1.4, "en-US", "1.4")]
[TestCase(1.4, "de-DE", "1,4")]
[TestCase(1400.01, "de-DE", "1400,01")]
[TestCase(1234.57, "ru-RU", "1234,57")]
[TestCase(1.094, "fr-FR", "1,094")]
[TestCase(1400.01, "zh-CN", "1400.01")]
public void TestRoundTrip(double value, string locale, string expected)
{
    CultureInfo.CurrentCulture = CultureInfo.GetCultureInfo(locale);

    var bindable = new BindableDouble(value);
    string? asString = bindable.ToString();
    Assert.AreEqual(expected, asString);
    Assert.DoesNotThrow(() => bindable.Parse(asString));
    Assert.AreEqual(value, bindable.Value, Precision.DOUBLE_EPSILON);
}
```

This will fail for French because `"1,094"` will get parsed in as `1094.0` and not `1.094`.

Reply:
Please note that this is a breaking change.

Reply:
Falling back to current culture - albeit buggy, error-prone, and unfortunate - also matches the rest of the c# ecosystem, so this probably is the more correct way to proceed.